modules/sampleteams.py

```diff
@@ -154,10 +154,10 @@
         },
         {
             "tier": "[Gen 7] Monotype",
-            "name": "Offensive Dragon",
-            "pokemon": ["dragonite", "latios", "kommo-o", "garchomp", "altaria-mega", "kyurem-black"],
-            "pokepasteURL": "https://pokepast.es/f950a52accdb3b92",
-            "username": "Eien",
+            "name": "Sticky Webs HO Rock",
+            "pokemon": ["shuckle", "golem-alola", "terrakion", "nihilego", "diancie-mega", "tyranitar"],
+            "pokepasteURL": "https://pokepast.es/e0e764ccf08f0b8c",
+            "username": "Tico 21",
```

Comment:
Why are you removing the old team?

Reply:
I have made a "pattern" to put only 6 teams in each tier, a user have made a request to me put him's team in the sample teams code, and I removed the Dragon type and added his Rock type team.

But I have a question, can the list be scrolled if have a lot teams? Like this:
![image](https://user-images.githubusercontent.com/73311080/97034200-e6d5f480-153a-11eb-9b57-850fa64ed751.png)

Reply:
No, but I could add code to put the teams into a dropdown (or you could).

Reply:
That's a much better solution than making arbitrary decision about the teams to include.

Reply:
That's looks cool! With this new feature we can add a lot teams, can u send me a printscreen with the model "dropdown"?

Reply:
<img width="670" alt="image" src="https://user-images.githubusercontent.com/56906084/97060530-746c1100-1548-11eb-9892-85cc57647cd2.png">
<img width="668" alt="image" src="https://user-images.githubusercontent.com/56906084/97060537-7afa8880-1548-11eb-83ef-8a8d249b02bc.png">

Reply:
(This is done.)

Reply:
I'm converting this pull request to a draft until this is resolved.